src/model/location_.ml

```diff
@@ -20,6 +20,9 @@ let at : span -> 'a -> 'a with_location = fun location value ->
 let location : 'a with_location -> span = fun {location; _} ->
   location
 
+let file_ext : span -> string = fun span ->
```

Comment:
Required to be able to determine the `from_lang` attribute... if `.mli` -> OCaml, if `.rei` -> Reason